src/firefly/js/visualize/ui/MultiItemViewerView.jsx

```diff
@@ -283,7 +283,7 @@ function computeTitleYOffset(sparseGridTitleLocation, itemPosTitle,rowIdx) {
     return titleOffset;
 }
 function computeDataYOffset(sparseGridTitleLocation, itemPosTitle,rowIdx) {
-    let titleOffset= sparseGridTitleLocation==='left' && itemPosTitle[0].hasData ? 0 : 30;
+    let titleOffset= sparseGridTitleLocation==='left' && itemPosTitle[0] && itemPosTitle[0].hasData ? 0 : 30;
```

Comment:
`itemPosTitle[0] && itemPosTitle[0].hasData`
can be replaced with 
get(itemPosTitle, '0.hasData')